lib/fathead/opam/parse.py

```diff
@@ -33,15 +32,15 @@ def get_contents(self):
     """
     The 'get_contents/0' member function loads the file into memory and
     reads the contents. The file buffer is then closed and then a
-    BeautifulSoup object is instanciated, with the contents. More on this
-    can be read at the BeautifulSoup documentation website.
+    BeautifulSoup object is instanciated, with the contents. More on
+    this can be read at the BeautifulSoup documentation website.
```

Comment:
@pjhampton Let's put [this link](https://www.crummy.com/software/BeautifulSoup/bs4/doc/) here.